src/goto-symex/build_goto_trace.cpp

```diff
@@ -161,7 +161,7 @@ static void update_internal_field(
     // "__CPROVER_*" function calls in __CPROVER_start are already marked as
     // internal. Don't mark any other function calls (i.e. "main"), function
     // arguments or any other parts of a code_function_callt as internal.
-    if(SSA_step.source.pc->code.get_statement()!=ID_function_call)
+    if(!SSA_step.is_function_call())
```

Comment:
I am not convinced those are equivalent - a function call (at the source) may have been replaced by an assert or assume.